ftw/publisher/sender/browser/views.py

```diff
@@ -318,7 +318,9 @@ def execute(self):
                                             jobCounter<self.batchsize):
 
             try:
-                if os.path.getsize(self.queue.nextJob().dataFile) == 0:
+                nextjob = self.queue.nextJob()
+                removed = nextjob.action == 'delete'
```

Comment:
@maethu I believe this change should not be necessary because we are no longer enqueuing `delete` jobs. Right?